File: gamepack/Decompiled/ClientError.java
public class ClientError extends RuntimeException {
   public static int au;
   public static int aw;
   public static String localPlayerName;
   String message;
   Throwable cause;

   ClientError(Throwable var1, String var2) {
      this.message = var2;
      this.cause = var1;
   }
}

File: find-names/src/main/java/mapper/identifiers/classes/ClientError.kt
package mapper.identifiers.classes

import mapper.abstractclasses.IdentityMapper
import mapper.predicateutilities.and
import mapper.predicateutilities.predicateOf
import mapper.predicateutilities.type
import mapper.wrappers.ClassWrapper
import mapper.wrappers.FieldWrapper

class ClientError : IdentityMapper.Class() {
    override val predicate = predicateOf<ClassWrapper> { it.superType == RuntimeException::class.type }
            .and { it.instanceFields.count { it.type == Throwable::class.type } == 1 }

    class cause : IdentityMapper.InstanceField() {
        override val predicate = predicateOf<FieldWrapper> { it.type == Throwable::class.type }
    }

    class message : IdentityMapper.InstanceField() {
        override val predicate = predicateOf<FieldWrapper> { it.type == String::class.type }
    }
}